csharp/src/Ice/ConnectionI.cs

```diff
@@ -2816,6 +2917,10 @@ internal void completed(LocalException ex)
 
     private int _upcallCount;
```

Comment:
Not related to the PR: Can you add a comment to clarify `_upcallCount`?

Reply:
Yes, done.